File: mobile/lib/models/game_dto.dart
class GameDto {
  final String gameId;
  String? playerX;
  String? playerO;

  GameDto({required this.gameId, this.playerX, this.playerO});

  factory GameDto.fromJson(Map<String, dynamic> json) {
    return GameDto(
      gameId: json['gameId'],
      playerX: json['playerX'],
      playerO: json['playerO'],
    );
  }
}

File: backend/TicTacToe/TicTacToe/DTO/GameDto.cs
namespace TicTacToe.DTO;

public class GameDto
{
    public Guid GameId { get; set; }

    public Guid? PlayerX { get; set; }  // null, если игрок X ещё не назначен

    public Guid? PlayerO { get; set; }  // null, если игрок O ещё не назначен
}